seisflows/workflow/forward.py

```diff
@@ -425,10 +425,20 @@ def evaluate_objective_function(self, save_residuals=False, **kwargs):
                          "objective function")
             return
 
+        if save_residuals:
+            save_residuals = save_residuals.format(src=self.solver.source_name,
+                                                   it="1")
```

Comment:
If we choose to keep the input string formatter general, then we will need an sc="0" here.